circinus/docker-compose.yml

```diff
@@ -22,12 +22,44 @@ services:
       local:
         aliases:
           - circinus-apache
+  mongodb:
+    container_name: delphinus-mongodb
+    image: mongo:4.0
+    environment:
+       - MONGO_DATA_DIR=/data/db
+       - MONGO_LOG_DIR=/dev/null
+    volumes:
+       - mongodbdata:/data/db 
+    ports:
+      - "127.0.0.1:${ENV_MONGO_PORT:-27017}:27017"
+    command: mongod 
+    networks:
+      local:
+        aliases:
+          - delphinus-mongodb
+
+  spring:
+    container_name: delphinus-spring
```

Comment:
Same here re: delphinus-named containers.

Reply:
Actually, they should be delphinus named since they are containers for the delphinus project inside the demo application.  The issue you are having is that you have delphinus running standalone alongisde the demo.  This is not a case that we should have in production.

Reply:
I completely closed and purged all my containers before running this compose for circinus and still encountered the issue.  I'll try again.

Reply:
I found the `delphinus` named containers in my docker and purged them - the compose runs, but I don't see the forwarding and virtualhost configuration working.  I see the page index, not the landing page.

Reply:
I had my .env configured with `apache-basic` and not `apache-basic-demo`.  Looks OK.